### PR TITLE
Add nkgrf17vel deformation model

### PR DESCRIPTION
**Pull request by kbevers**
Supporting grid file for the [2020 NKG transformations](https://github.com/OSGeo/PROJ/pull/2494).

I had trouble regenerating `index.html` (I can't get a recent enough proj/gdal combo with conda it seems), so hopefully I can get a bit of help with that :-)

### CODE REVIEW

File: eur_nkg/eur_nkg_README.txt [hunk 1]
```diff
@@ -18,6 +18,18 @@ examples of use.
 
 * eur_nkg_nkgrf03vel_realigned.tif
 
+### Nordic+Baltic countries: 2017 Deformation model
+
+*Source*: [The Nordic Geodetic Commission](https://http://www.nordicgeodeticcommission.com/)
```
Comment (rouault) [inline eur_nkg/eur_nkg_README.txt:23]: For odd formatting reason, you need to add two spaces at the end of that line and the following one. See how it renders currently: https://github.com/OSGeo/PROJ-data/tree/1345aa3b88616bb8beb63b5cf73cc0506dc4befa/eur_nkg

**Review by rouault — COMMENTED** [review]
The following warnings are emitted by check_gtiff_grid.py:
- Metadata AREA_OF_USE=Nordic and Baltic countries ignored
- Metadata DESCRIPTION=NKG_RF17vel model ignored

The first one is due to the fact that what is expected (and used in other grids) is ``area_of_use`` in lowercase.
The second one may be ignored.

**Comment by kbevers** [comment]
> Metadata AREA_OF_USE=Nordic and Baltic countries ignored

Is there an easy way to fix that using GDAL tools? Perhaps by using gdal_translate and a vrt file? I am afraid I haven't got the source material and script readily available since the grid was processed by a colleague.

File: eur_nkg/eur_nkg_README.txt (hunk 1 above)
Comment (kbevers) [reply eur_nkg/eur_nkg_README.txt:23]: Right, yes. Should be fixed now.

**Comment by rouault** [comment]
> Perhaps by using gdal_translate and a vrt file?

yes, that's probably the best option here. Make sure you re-apply the compression options -co COMPRESS=DEFLATE -co PREDICTOR=3 when converting the VRT to TIFF. And run grid_tools/cloud_optimize_gtiff.py afterwards.
Well, an easier solution, as it is just a change of case, could be to use an hexadecimal editor (I use "bless" on Linux) and just change it :-).
Don't loose time on that. I'll do it for you right now

**Comment by kbevers** [comment]
> Don't loose time on that. I'll do it for you right now

Thanks! I was just reading up on `gdal_edit` which also seems to be able to do the job but I am much more confident in you than me :)

**Comment by rouault** [comment]
> `gdal_edit`

To add new metadata item yes, but to rename an existing one, it isn't really helpful.

> I'll do it for you right now

Done, and index regenerated

**Comment by kbevers** [comment]
> To add new metadata item yes, but to rename an existing one, it isn't really helpful.

I figured using `-unsetmd` in combination with a bunch of `-mo`'s would do the trick as well. But I am sure you have better ways to do it!

> Done, and index regenerated

Thanks, very appreciated!

**Comment by rouault** [comment]
> I figured using -unsetmd in combination with a bunch of -mo's would do the trick 

ah, yes you're right. That could have done it. The hex editor was just faster for that very particular use case :-)

**Comment by rouault** [comment]
Uploaded to CDN

**Pull request merged**